test/test_jit_cuda_fuser.py

```diff
@@ -598,7 +598,9 @@ def t(x: torch.Tensor, y: torch.Tensor):
             #  bfloat16 kernels instead of eager mode
             #  implementation, since mismatch in cast
             #  adds excessive noise.
-            o = t(x.to(torch.float64), y.to(torch.float64)).to(torch.bfloat16)
+            o = t(x.to(torch.float64), y.to(torch.float64))
+            if o.dtype == torch.float64:
```

Comment:
nitpick, This seems to be different from the description in the PR.

we only cast to bf16 when output is float64 here, while the PR description suggests that we cast to bf16 when output is in floating type, for which we should probably use `o.dtype.is_floating_point`

Reply:
updated Thanks!